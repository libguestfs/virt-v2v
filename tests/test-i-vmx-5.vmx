.encoding = "UTF-8"
config.version = "8"
virtualHW.version = "11"
memsize = "2048"
MemAllowAutoScaleDown = "FALSE"
displayName = "MSEdge - Win10_preview"
guestOS = "windows9-64"
cpuid.coresPerSocket = "1"
vmci0.present = "TRUE"
ethernet0.present = "TRUE"
ethernet0.addressType = "generated"
ethernet0.connectionType = "bridged"
ethernet0.virtualDev = "e1000e"
ethernet0.startConnected = "TRUE"
scsi0.present = "TRUE"
scsi0.virtualDev = "lsisas1068"
scsi0:0.present = "TRUE"
scsi0:0.fileName = "MSEdge-Win10_preview.vmdk"
bios.bootorder = "hdd,CDROM"
cleanshutdown = "TRUE"
ethernet0.bsdname = "en0"
ethernet0.displayname = "Ethernet"
ethernet0.linkstatepropagation.enable = "FALSE"
gui.fullscreenatpoweron = "FALSE"
gui.viewmodeatpoweron = "windowed"
hgfs.linkrootshare = "TRUE"
hgfs.maprootshare = "TRUE"
isolation.tools.hgfs.disable = "FALSE"
monitor.phys_bits_used = "42"
msg.autoanswer = "true"
numa.autosize.cookie = "10001"
numa.autosize.vcpu.maxpervirtualnode = "1"
nvram = "MSEdge - Win10_preview.nvram"
proxyapps.publishtohost = "FALSE"
remotedisplay.vnc.enabled = "FALSE"
remotedisplay.vnc.port = "5900"
replay.filename = ""
replay.supported = "FALSE"
scsi0:0.redo = ""
softpoweroff = "TRUE"
tools.synctime = "true"
uuid.action = "create"
virtualhw.productcompatibility = "hosted"
vm.genid = "-570734802784577186"
vm.genidx = "-5042519231342505152"
vmotion.checkpointfbsize = "33554432"
pciBridge0.present = "TRUE"
tools.upgrade.policy = "useGlobal"
pciBridge4.present = "TRUE"
pciBridge4.virtualDev = "pcieRootPort"
pciBridge4.functions = "8"
pciBridge5.present = "TRUE"
pciBridge5.virtualDev = "pcieRootPort"
pciBridge5.functions = "8"
pciBridge6.present = "TRUE"
pciBridge6.virtualDev = "pcieRootPort"
pciBridge6.functions = "8"
pciBridge7.present = "TRUE"
pciBridge7.virtualDev = "pcieRootPort"
pciBridge7.functions = "8"
hpet0.present = "TRUE"
extendedConfigFile = "MSEdge - Win10_preview.vmxf"
uuid.bios = "56 4d 54 59 5d 0c 51 8d-d3 40 38 4f fe bf e4 5d"
uuid.location = "56 4d 54 59 5d 0c 51 8d-d3 40 38 4f fe bf e4 5d"
migrate.hostlog = ".\MSEdge - Win10_preview-aee69569.hlog"
pciBridge0.pciSlotNumber = "17"
pciBridge4.pciSlotNumber = "21"
pciBridge5.pciSlotNumber = "22"
pciBridge6.pciSlotNumber = "23"
pciBridge7.pciSlotNumber = "24"
scsi0.pciSlotNumber = "160"
ethernet0.pciSlotNumber = "192"
vmci0.pciSlotNumber = "32"
scsi0.sasWWID = "50 05 05 69 5d 0c 51 80"
ethernet0.generatedAddress = "00:0c:29:bf:e4:5d"
ethernet0.generatedAddressOffset = "0"
vmci0.id = "-20978595"
vmotion.checkpointSVGAPrimarySize = "33554432"
toolsInstallManager.updateCounter = "1"
toolsInstallManager.lastInstallError = "21004"
