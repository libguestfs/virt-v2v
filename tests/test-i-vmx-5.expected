
Source guest information (--print-source option):

    source name: MSEdge - Win10_preview
hypervisor type: vmware
       VM genid: 8312995e-57b8-f814-4013-c1ba795f05ba
         memory: 2147483648 (bytes)
       nr vCPUs: 1
     CPU vendor: 
      CPU model: 
   CPU topology: sockets: 1 cores/socket: 1 threads/core: 1
   CPU features: 
       firmware: bios
        display: 
          sound: 
disks:
	0 [scsi]
removable media:

NICs:
	Bridge "ethernet0" mac: 00:0c:29:bf:e4:5d [e1000e]

